infra/k8s/README.md

```diff
@@ -81,63 +85,23 @@ kops create cluster \
     - net.core.somaxconn
 ```
 
-6. Create cluster
-```
-kops create -f cluster.yaml
-kops create secret --name $NAME sshpublickey admin -i ~/.ssh/id_rsa.pub
-kops update cluster $NAME --yes
-```
-
-7. Wait for all nodes to appear in `kubectl get nodes` with `Ready` state, and for all pods in `kube-system` namespace to be `Running`.
-```
-watch 'kubectl get nodes -o wide'
-
-kubectl -n kube-system get pods -o wide
-```
-
-8. Add AWS Assets S3 bucket secrets to Kubernetes
-```
-kubectl create secret generic assets-s3-bucket --from-literal=access-key="$ASSETS_ACCESS_KEY" \
-                                               --from-literal=secret-key="$ASSETS_SECRET_KEY" \
-                                               --from-literal=bucket-name="$ASSETS_BUCKET_NAME"
-```
-
-8. Install CNI-Genie, Weave and S3 bucket daemonset
-
-```
-kubectl apply -f ./infra/k8s/kops-weave/genie-plugin.yaml \
-              -f ./infra/k8s/kops-weave/weave.yml \
-              -f ./infra/k8s/kops-weave/s3bucket.yml
-```
-
-9. Destroy the cluster when you're done working on it
-
-```
-kops delete cluster $NAME --yes
-```
-
-
-## Setup Testground remote dependencies on your cluster
-
-1. Set up Helm
+6. Set up Helm and add the `stable` Helm Charts repository
 
 ```
 helm repo add stable https://kubernetes-charts.storage.googleapis.com/
 helm repo update
 ```
 
-2. Create a `Redis` service on your Kubernetes cluster
-
+7. Apply cluster setup and install Testground dependencies
 ```
-helm install redis stable/redis --values ./infra/k8s/redis-values.yaml
+./install.sh $NAME $CLUSTER_SPEC $PUBKEY $WORKER_NODES
```

Comment:
Note that this file is not idempotent. Then again it is fairly straight-forward. In case it has to become any more elaborate than it already is, we might have to consider other tools than `bash`.